src/style.css

```diff
@@ -1 +1,86 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+body {
+  margin-top: 0;
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+.input {
+  display: block;
+  border: 1px solid #e1e7ed;
+  color: #3d4e61;
+  font-weight: 900;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  font-size: 18px;
+  border-radius: 3px;
+  outline: none;
+  box-sizing: border-box;
+  width: 100%;
+  padding-bottom: 4px;
+  margin-top: 20px;
+}
+
+.input::placeholder {
+  font-weight: 300;
+  text-shadow: none;
+  color: #3d4e61;
+}
+
+.big-input {
+  height: 70px;
+  padding-left: 61px;
+  font-size: 16px;
+  background-size: 19px 19px;
+  background-position: 25px calc(50% - 1px);
+  text-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
+
+.small-input {
+  height: 42px;
+  padding-left: 33px;
+  background-size: 11px 11px;
+  background-position: 12px calc(50% - 1px);
+  font-size: 14px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
+
+.big-input::placeholder {
+  font-size: 15px;
+  letter-spacing: 0.7px;
+}
+
+.small-input::placeholder {
+  font-size: 13px;
+  letter-spacing: 0.4px;
+}
+
+.big-input:hover {
+  box-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+}
+
+.big-input:focus {
+  outline: none;
+  background-image: url(images/Search.svg), linear-gradient(#fff, #f6f6f7);
```

Comment:
Ensure that the image path for the `background-image` in the focus state of `.big-input` is correct and accessible. It should be `./images/Search.svg` to match the other image paths.